src/qml/components/Footer.qml

```diff
@@ -5,12 +5,13 @@ import QtQuick.Controls 2.12
 import "../theme"
 
 Item {
+    property string _text: ""
```

Comment:
Use aliases where it's possible. There is no need to underline variable

Reply:
Ok